src/lib/helpers.ts

```diff
@@ -126,14 +126,20 @@ export class Deferred<T = any> {
     })
   }
 }
+
 // Taken from: https://stackoverflow.com/questions/38552003/how-to-decode-jwt-token-in-javascript-without-using-a-library
 export function decodeJWTPayload(token: string) {
+  const base64UrlRegex = /^([a-z0-9_-]{4})*($|[a-z0-9_-]{3}=?$|[a-z0-9_-]{2}(==)?$)$/i
```

Comment:
would be great to add a comment here on what this regex checks for, does this expression just check for base64Url?

Reply:
Yes it does. Believe I got that from someone else on the team a while back. I'll add a comment.